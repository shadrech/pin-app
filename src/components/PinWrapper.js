import React from "react";
import styled from "react-emotion";

import DisplayWrapper from "./DisplayWrapper";
import ButtonsWrapper from "./ButtonsWrapper";

const Wrapper = styled('div')`
    height: 30rem;
    width: 18rem;
    background: white;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    padding: 2rem 0 2rem 0;
`;

const PinWrapper = () => (
    <Wrapper>
        <DisplayWrapper />
        <ButtonsWrapper />
    </Wrapper>
);

export default PinWrapper;