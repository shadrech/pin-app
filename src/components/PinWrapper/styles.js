import styled from "react-emotion";

export const Wrapper = styled('div')`
    height: 30rem;
    width: 18rem;
    background: white;
    display: flex;
    flex-direction: column;
    justify-content: space-evenly;
    padding: 2rem 0 1rem 0;
`;