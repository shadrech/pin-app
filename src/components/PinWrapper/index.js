import React from "react";

import { Wrapper } from "./styles";
import DisplayWrapper from "../DisplayWrapper";
import ButtonsWrapper from "../ButtonsWrapper";
import Lock from "../Lock";

const PinWrapper = () => (
    <Wrapper>
        <Lock />
        <DisplayWrapper />
        <ButtonsWrapper />
    </Wrapper>
);

export default PinWrapper;