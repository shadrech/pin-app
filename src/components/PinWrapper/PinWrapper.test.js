import React from "react";
import { shallow } from "enzyme";
import PinWrapper from ".";

describe("<PinWrapper /> component", () => {    
    test("should match snapshot", () => {
        const wrapper = shallow(<PinWrapper />);
        expect(wrapper).toMatchSnapshot();
    })
})