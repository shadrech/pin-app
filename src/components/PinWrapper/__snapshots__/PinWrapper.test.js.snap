// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PinWrapper /> component should match snapshot 1`] = `
<Styled(div)>
  <Connect(Lock) />
  <Connect(DisplayWrapper) />
  <Connect(ButtonsWrapper) />
</Styled(div)>
`;
